testing/PIMS.Tests.Automation/PageObjects/PropertyInformation.cs

```diff
@@ -237,7 +237,72 @@ public void CancelPropertyDetails()
 
         public void UpdateMinPropertyDetails(string notes)
         {
-            WaitUntilClickable(propertyDetailsNotesTextarea);
+            Wait();
+            ClearInput(propertyDetailsNotesTextarea);
+            webDriver.FindElement(propertyDetailsNotesTextarea).SendKeys(notes);
+        }
+
+        //To-Do - Erase after updating AcquisitionFiles
+        public void UpdateMaxPropertyDetails(string addressLine1, string addressLine2, string city, string postalCode, string municipalZoning, string sqMts, string cubeMts, string notes)
+        {
+
+            Wait(5000);
```

Comment:
question - I'm not sure we update these integration test files. That usually comes from @stairaku  - are you sure this change is related/required for this PR?